### PR TITLE
feat: upgrade Zinnia to v0.17.0

### PR DESCRIPTION
**Pull request by bajtos**
~~CI is failing because Zinnia's release process hasn't finished yet: https://github.com/filecoin-station/zinnia/actions/runs/7802847939~~

### CODE REVIEW

File: lib/zinnia.js [hunk 1]
```diff
@@ -283,6 +283,13 @@ async function handleEvents ({
     try {
       const event = JSON.parse(line)
       switch (event.type) {
+        case 'activity:started':
+          onActivity({
+            type: 'info',
+            message: `${capitalize(module)} started`,
```
Comment (bajtos) [inline lib/zinnia.js:289]: Do you think it's worth writing a new test for this? Any suggestions one how to write it?

Maybe this is already covered by the existing test here:

https://github.com/filecoin-station/core/blob/f77317966725f110151bda2edb443c0186aa0919/test/station.js#L15-L16

Thoughts?

Comment (bajtos) [reply lib/zinnia.js:289]: The tests are failing when I comment out the `onActivity` call - I think we are good here.

```js
        case 'activity:started':
          // onActivity({
          //   type: 'info',
          //   message: `${capitalize(module)} started`,
          //   source: module
          // })
          break
```

Test output:

```
  8 passing (48s)
  3 failing

  1) Station
       runs Zinnia:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)

  2) Station
       outputs events:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)

  3) Station
       outputs events json:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)
```

Comment (juliangruber) [reply lib/zinnia.js:289]: sounds good as it is yeah!

**Review by juliangruber — APPROVED** [review]

**Comment by juliangruber** [comment]
https://github.com/filecoin-station/core/releases/tag/v20.0.0

**Pull request merged**